book/10-git-internals/sections/plumbing-porcelain.asc

```diff
@@ -17,21 +17,22 @@ Here's what it looks like:
 [source,console]
 ----
 $ ls -F1
-HEAD
-config*
+branches/
+config
 description
+HEAD
 hooks/
 info/
 objects/
 refs/
```

Comment:
Interesting. Where are you seeing a `branches` folder? Here's the output from my machine:

```
$ git init foo
Initialized empty Git repository in /private/tmp/foo/.git/
$ cd foo/.git
$ ls -F1
HEAD
config
description
hooks/
info/
objects/
refs/
$ git --version
git version 2.14.1
```

Reply:
I just created a new repo on my Fedora 26 box (git --version = 2.13.6), and I get a branches/ folder even in a new repo. Are you saying you don't?